### PR TITLE
Add new package: libmcrypt

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

**Review by alalazo — CHANGES_REQUESTED** [review]
Failing style test on CI

File: var/spack/repos/builtin/packages/libmcrypt/package.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+class Libmcrypt(AutotoolsPackage):
+    """Libmcrypt is a thread-safe library providing a uniform
+    interface to access several block and stream encryption
+    algorithms."""
+
+    homepage = "https://sourceforge.net/projects/mcrypt/files/Libmcrypt/"
+    url      = "https://sourceforge.net/projects/mcrypt/files/Libmcrypt/2.5.8/libmcrypt-2.5.8.tar.gz"
+
+    version('2.5.8', sha256='e4eb6c074bbab168ac47b947c195ff8cef9d51a211cdd18ca9c9ef34d27a373e')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/libmcrypt/package.py:17]: Latest version seems to be 2.6.8

**Review by alalazo — CHANGES_REQUESTED** [review]
Can you add the latest version too?

File: var/spack/repos/builtin/packages/libmcrypt/package.py (hunk 1 above)
Comment (darmac) [reply var/spack/repos/builtin/packages/libmcrypt/package.py:17]: I can only find `2.5.8` in sourceforge
https://sourceforge.net/projects/mcrypt/files/Libmcrypt/

Comment (alalazo) [reply var/spack/repos/builtin/packages/libmcrypt/package.py:17]: Screenshot from 2021-03-21 09-14-33

Comment (darmac) [reply var/spack/repos/builtin/packages/libmcrypt/package.py:17]: `mcrypt` is not `libmcrypt`, they are two softwares~

**Review by alalazo — APPROVED** [review]

**Pull request merged**